core/src/epicli/data/common/ansible/playbooks/roles/postgresql/tasks/configure-postgresql.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Set OS specific paths
+  set_fact:
+    pg_config_dir: "{{ pg_config_dir[ansible_os_family] }}"
+  vars:
+    pg_config_dir:
```

Comment:
Cannot this be moved to defaults yaml for PostgreSQL role?

Reply:
I don't think that adding rather "static" options (paths depend on installed packages) to the configuration is a good idea unless someone needs them.